src/schema/objects/metadata.yaml

```diff
@@ -1983,9 +2036,19 @@ RecordingType:
     interest (for example, stimulus presentations or subject responses).
   type: string
   enum:
-  - continuous
-  - epoched
-  - discontinuous
+  - continuous:
```

Comment:
I would prefer to have meaningful descriptions for the continuous and discontinuous values.